sql/1685202737.sql

```diff
@@ -0,0 +1,13 @@
+-- +migrate Up
+ALTER TABLE convoy.portal_links 
```

Comment:
Do you think we'd need to add an index for the `owner_id` field? Seeing as we would be using it in queries